src/Data/String.purs

```diff
@@ -222,16 +232,16 @@ foreign import take :: Int -> String -> String
 -- | Returns the string without the first `n` characters.
 foreign import drop :: Int -> String -> String
 
--- | Returns the number of contiguous characters at the beginning
--- | of the string for which the predicate holds.
+-- | Returns the number of contiguous characters *at the beginning
+-- | of the string* for which the predicate holds.
```

Comment:
I'm not hugely keen on this change, personally, I think it's clear enough already without needing to draw extra attention to the fact that it's counting from the beginning.